### ISSUE TITLE
Consider ligatures for `=~` and `!~` similar to FiraCode

### ISSUE THREAD

**Issue by phene** [rubjo/victor-mono#153]
Having ligatures for Ruby's Regex operators `=~` and `!~` would be amazing. See FiraCode's for an example: https://github.com/tonsky/FiraCode/blob/master/extras/ligatures.png

**Comment by rubjo** [comment]
Thanks!

Schedule is absolutely packed but yes, I’ll include them for the next release. 🙂

**Comment by rubjo** [comment]
Released in 1.5.5.

**Comment by galou** [comment]
The `=~` ligature is rather unfortunate in bash e.g. when defining an environment variable (`MY_PRG_HOME=~/.config/my_prg`), where the two symbols should be definitely separated.

**Comment by rubjo** [comment]
I agree. Taking a note for the future, don't know if I'm going to change the font. Maybe disable ligatures in your terminal if possible.
